docs/source/install.rst

```diff
@@ -0,0 +1,204 @@
+|                                           | in your local directory. |
+|                                           | It is assumed the        |
+|                                           | dependencies have been   |
+|                                           | built.                   |
++-------------------------------------------+--------------------------+
+| ``./install.sh -idc``                     | Build library            |
+|                                           | dependencies, client     |
+|                                           | dependencies, library,   |
+|                                           | client, then build and   |
+|                                           | install the hipBLAS      |
+|                                           | package. You will be     |
+|                                           | prompted for sudo        |
+|                                           | access. It is expected   |
+|                                           | that if you want to      |
+|                                           | install for all users    |
+|                                           | you use the -i flag. If  |
+|                                           | you want to keep hipBLAS |
+|                                           | in your local directory, |
+|                                           | you do not need the -i   |
+|                                           | flag.                    |
++-------------------------------------------+--------------------------+
+| ``./install.sh -ic``                      | Build and install        |
+|                                           | hipBLAS package, and     |
+|                                           | build the client. You    |
+|                                           | will be prompted for     |
+|                                           | sudo access. This will   |
+|                                           | install for all users.   |
+|                                           | If you want to keep      |
+|                                           | hipBLAS in your local    |
+|                                           | directory, you do not    |
+|                                           | need the -i flag.        |
++-------------------------------------------+--------------------------+
+
+Dependencies
+============
+
+Dependencies are listed in the script install.sh. Use ``install.sh`` with ``-d`` option to install dependencies.
+CMake has a minimum version requirement listed in the file install.sh. See --cmake_install flag in install.sh to upgrade automatically.
+
+
+Manual build (all supported platforms)
+=======================================
+
+This section has useful information on how to configure cmake and manually build.
+
+Dependencies For Building Library
+---------------------------------
+
+CMake 3.5 or later
+~~~~~~~~~~~~~~~~~~
```

Comment:
remove this CMake 3.5 section. PR#427 has code to check that the user has CMake version >= 3.16.8. This CMake 3.5 section is out of date.